projects/ngx-powerbi/package.json

```diff
@@ -24,8 +27,9 @@
     "reporting"
   ],
   "peerDependencies": {
-    "@angular/common": "^7.0.0",
-    "@angular/core": "^7.0.0",
-    "powerbi-client": "^2.6.5"
+    "@angular/common": "^9.0.6",
+    "@angular/core": "^9.0.6",
+    "powerbi-client": "^2.6.5",
```

Comment:
Maybe it makes sense to update this to 2.11.0

Reply:
I'll look into it.

Reply:
I'm using in my project 2.11.0 and it works fine. I updated it.